src/zac/core/forms.py

```diff
@@ -287,3 +286,34 @@ def get_process_variables(self) -> Dict[str, List[str]]:
             "documenten": self.cleaned_data["documenten"],
             "kownslReviewRequestId": self.cleaned_data["review_request"],
         }
+
+    def on_submission(self):
+        if self._review_type is not None:
+            review_request = create_review_request(
+                self.zaak_url,
+                review_type=self._review_type,
+                num_assigned_users=len(self.cleaned_data["users"]),
+            )
+            self.cleaned_data["review_request"] = str(review_request.id)
+        else:
+            raise NotImplementedError(
+                f"Variable _review_type needs to be defined in {self.__name__}"
+            )
+
+
+class ConfigureAdviceRequestForm(ConfigureReviewRequestForm):
+    """
+    Child class of ConfigureReviewRequestForm which specifies that the type of review requested
```

Comment:
Docstrings should follow the format of:

```
Do some action to achieve something.

More elaborate explanation of what's happening, attribute documentation... etc.
```

Suggested docstring:
```suggestion
    Create an "advice" type of review request on valid submission.
```